.eslintrc.js

```diff
@@ -1,25 +1,42 @@
 module.exports = {
   extends: 'airbnb',
+  env: { browser: true },
   rules: {
+    // ESLint ----------------------------------
+    'arrow-parens': ['error', 'always'],
+    'block-spacing': 0,
+    'global-require': 0,
     'max-len': [1, 160, 4, {"ignoreUrls": true}],
     'no-console': 0,
+    'no-mixed-operators': 0,
     'no-multi-spaces': [2, { exceptions: { "ImportDeclaration": true } }],
     'no-nested-ternary': 0,
     'no-param-reassign': [2, { props: false }],
+    'no-plusplus': ["error", { "allowForLoopAfterthoughts": true }],
+    'no-prototype-builtins': 0,
+    'no-underscore-dangle': 0,
     'no-unused-vars': [2, { args: 'none' }],
     'no-var': 0,
-    'no-underscore-dangle': 0,
     'one-var': 0,
-    'block-spacing': 0,
-    'global-require': 0,
+    // imports ----------------------------------
+    'import/imports-first': 0,
+    'import/no-extraneous-dependencies': 0,
+    'import/no-named-as-default-member': 0,
     'import/no-unresolved': 0,
-    'react/no-is-mounted': 1,
-    'react/prefer-es6-class': 0,
-    'react/prefer-stateless-function': 0,
+    'import/prefer-default-export': 0,
+    // react ----------------------------------
+    'react/forbid-prop-types': 0,
+    'react/jsx-closing-bracket-location': 0,
```

Comment:
That's been there for some time. 
https://github.com/Kitware/HPCCloud/blame/master/.eslintrc.js#L22

As long as proper indentation is enforced adding a new line isn't necessary to distinguish elements.

```html
<div>
  <MyDiv attributes that are very long and take up many 
  lines of text />
  <NextDiv />
</div>
```